.coveragerc

```diff
@@ -1,6 +1,8 @@
 [run]
 branch = True
 source = zope.interface
+# New in 5.0; required for the GHA coveralls submission.
+relative_files = True
```

Comment:
Hmm, I've needed it in every one, at least when using 5.0 and the action (instead of manually calling the `coveralls` command, which I found problematic). Perhaps one of those is different?

Reply:
(The [action documentation](https://github.com/AndreMiras/coveralls-python-action) specifies this setting.)

Reply:
This could be it, I've always been using `pip install coveralls && coveralls`.  (Last time I checked, which was some time ago, the official Coveralls action didn't support Python.)